deployment/ansible/roles/cac-tripplanner.nginx/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Tasks and related artifacts copied from azavea.nginx role
+# Customized to use Nginx Stable PPA that works with Ubuntu 22.04
+- name: Configure the Nginx PPA
+  apt_repository: repo=ppa:ondrej/nginx state=present
```

Comment:
Hard to tell since it's a new file, but this line has been changed from `apt_repository: repo=ppa:nginx/{{ nginx_version }} state=present`.